src/app/views/seminars/seminars.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="container-fluid mt-4 px-5">
+  <div class="row">
+    <div class="col-md-6">
+      <h2><b>Seminars</b></h2>
+    </div>
+    <div class="col-md-6">
+      <div class="d-flex flex-wrap justify-content-end">
+        <button id="GuestSeminarsBtn" class="btn me-2" (click)="updateData('GuestSeminars')" [class.selected]="selectedTab === 'GuestSeminars'">
+          Guest Seminars ({{seminars.ICWaR_Guest_Seminar_Series.length}})
+        </button>
+        <button id="CommunitySeminarsBtn" class="btn me-2" (click)="updateData('CommunitySeminars')" [class.selected]="selectedTab === 'CommunitySeminars'">
+          ICWaR Community Seminars ({{seminars.ICWaR_Community_Seminar_Series.length}})
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+<hr>
+
+<!-- Guest Seminars Section -->
+<div id="GuestSeminars" class="p-4 row d-flex justify-content-center col-12 pt-0" *ngIf="selectedTab === 'GuestSeminars'">
+  <ng-container *ngFor="let item of seminars.ICWaR_Guest_Seminar_Series; index as index">
+    <div class="card-bg shadow border border-1 mb-2 card col-12 col-md-10 border-dark p-3">
+      <div class="card-text row">
+        <div class="col-md-9 col-12 mb-2">
```

Comment:
For DateTime d should be small . Follow Camle casing. it should be item.dateTime.
Fix this throughout your changes